airflow/providers/snowflake/hooks/snowflake.py

```diff
@@ -25,10 +25,22 @@
 from snowflake import connector
 from snowflake.connector import DictCursor, SnowflakeConnection
 from snowflake.connector.util_text import split_statements
+from sqlalchemy import create_engine
 
 from airflow.hooks.dbapi import DbApiHook
 
 
+def _boolify(value):
+    if isinstance(value, bool):
+        return value
+    if isinstance(value, str):
+        if value.lower() == 'false':
+            return False
+        elif value.lower() == 'true':
+            return True
+    return value
```

Comment:
I'm not sure about the return - what it means it terms of boolean value?

Reply:
If it can be converted to a bool value then a conversation is performed, but if an incorrect value is passed it is passed unchanged and then the layer below can handle it.

Reply:
I changed the behavior and for an invalid value an exception is now raised

Reply:
Did you not push the change? The implementation here does not raise, from what I can tell.

Reply:
Also we have `airflow.utils.strings.to_boolean()`. Its behaviour is different from the one you have here, but might be more expected since the function is used in other parts of Airflow.

Reply:
Pushed now

Reply:
`to_boolean` looks interesting. Updated and. pushed.